frontends/web/src/routes/device/bitbox02/setdevicename.tsx

```diff
@@ -45,9 +45,15 @@ export const SetDeviceName = ({
   const updateName = async () => {
     setInProgress(true);
     try {
-      await setDeviceName(deviceID, name);
```

Comment:
before (here) if it fails OR throws an error, it would catch and show alertuser error.

now I added success checks and manually throw new Error so that alertUser does not need to be written in 3 places, but seems a bit ugly.

Reply:
Looks okay to me, but it'll still always show 'Device name could not be set' regardless of the error being thrown, right?